scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram.py

```diff
@@ -151,7 +152,11 @@ def beam_search_eval(
     model.cfg.decoding.beam = cfg.decoding
 
     # Update model's decoding strategy
-    model.change_decoding_strategy(model.cfg.decoding)
+    if isinstance(model, EncDecHybridRNNTCTCModel):
+        model.change_decoding_strategy(model.cfg.decoding, decoder_type='ctc')
+        model.decoding = model.ctc_decoding
```

Comment:
Don't in place replace like this, use the change decoding strategy above

Reply:
moved it into change_decoding_strategy()